### DISCUSSION TITLE
Hide card if no events planned

### DISCUSSION THREAD

**Discussion by Bluscream** [totaldebug/atomic-calendar-revive#1516] [Feature Requests] [answered]
It would be nice if there was a way to make the card hidden if no events are found in the specified period

![opera_igKBczCpWX](https://github.com/totaldebug/atomic-calendar-revive/assets/3318223/8ae53f24-6a42-4e05-86ce-b2bf12f0cd48)

**Comment by marksie1988** [comment, marked as answer]
This has been implemented on the latest beta, will be released soon
